tests/AutoReview/FixerFactoryTest.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\AutoReview;
+
+use PhpCsFixer\Fixer\FixerInterface;
+use PhpCsFixer\FixerFactory;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ *
+ * @coversNothing
+ * @group auto-review
+ */
+final class FixerFactoryTest extends \PHPUnit_Framework_TestCase
```

Comment:
-1 for the merge. I agree some test here are not about the `FixerFactory` and should be moved to a better test some day, but the `FixerTest` is not the right spot either IMHO. Anyway that move does not need to be part of this PR I think.